velox/functions/prestosql/Repeat.cpp

```diff
@@ -73,14 +79,16 @@ class RepeatFunction : public exec::VectorFunction {
       return BaseVector::createNullConstant(outputType, numRows, pool);
     }
 
-    const auto count = constantCount->valueAt(0);
+    auto count = constantCount->valueAt(0);
     try {
       checkCount(count);
     } catch (const VeloxUserError&) {
       context.setErrors(rows, std::current_exception());
       return nullptr;
     }
-
+    if (count < 0) {
```

Comment:
nit: perhaps, move this logic into checkCount and have checkCount return possibly modified value.